File: example/scalalib/config/2-testing/integration/src/QuxTests.scala
package qux
import utest.*
object QuxIntegrationTests extends TestSuite {
  def tests = Tests {
    test("helloworld") {
      val result = Qux.hello()
      QuxTests.assertHello(result)
      QuxTests.assertWorld(result)
      result
    }
  }
}

File: example/kotlinlib/testing/1-testing/test/src/qux/QuxTests.kt
package qux

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.string.shouldEndWith
import io.kotest.matchers.string.shouldStartWith

fun assertHello(result: String) {
    result shouldStartWith "Hello"
}

fun assertWorld(result: String) {
    result shouldEndWith "World"
}

class QuxTests :
    FunSpec({

        test("hello") {
            val result = Qux.hello()
            assertHello(result)
        }

        test("world") {
            val result = Qux.hello()
            assertWorld(result)
        }
    })
